semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FStart.html

```diff
@@ -392,7 +391,7 @@
     {{#if selectedDomain}}
       <div class='KeywordTypeQuery--domainCards'>
       <div data-flex-layout='rows stretch-stretch'>
-        {{#each domains}}
+        {{#each (array-sortBy domains (func (d) d.thumbnail) descending=false)}}
```

Comment:
how does this behave if at some point there is a thumbail missing in the configuration?

Reply:
icons are part of `semopenalex` app and for pragmatic approach, for now, i made sure thumbnail/icons are presented.